src/styles/card.scss

```diff
@@ -0,0 +1,140 @@
+@use './variables' as *;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  margin: 0 auto;
+
+  @media (min-width: 0) {
+    & {
+      grid-template-columns: repeat(1, 200px);
+    }
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, 200px);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  grid-auto-flow: column;
+  transition: 0.3s ease-in;
```

Comment:
The 'grid-auto-flow' property does not seem relevant to the '.card' class since it is not a grid container. This property is typically used on grid containers to control the placement of grid items, so it should be removed if not necessary.